includes/specials/SpecialAllmessages.php

```diff
@@ -291,7 +291,7 @@ public static function getCustomisedStatuses( $messageNames, $langcode = 'en', $
 	 *  This function normally does a database query to get the results; we need
 	 * to make a pretend result using a FakeResultWrapper.
 	 */
-	function reallyDoQuery( $offset, $limit, $descending ) {
+	function reallyDoQuery( $offset, $limit, $descending, $info = [] ) {
```

Comment:
Is this needed here?

Reply:
Same as above basically.

Reply:
@TK-999 — yes, as child and parent methods declarations have to be identical.